docker/puppetserver/release.Dockerfile

```diff
@@ -1,8 +1,10 @@
-ARG vcs_ref
-ARG build_date
 ARG version="6.0.0"
 ARG namespace="puppet"
 FROM "$namespace"/puppetserver-base:"$version"
+ARG vcs_ref
```

Comment:
Since there's only one stage in this file, why not also move the `version` and `namespace` values below the `FROM`?

Reply:
because those are used in the `FROM` line

Reply:
Hahahaha, I suppose that's a good reason! Sheesh.